flowmachine/flowmachine/features/network/total_network_objects.py

```diff
@@ -203,7 +203,7 @@ def aggregate(self, by=None, statistic="avg"):
         )
 
 
-class AggregateNetworkObjects(GeoDataMixin, Query):
+class AggregateNetworkObjects(TotalNetworkObjects, Query):
```

Comment:
What's going on here?

Reply:
This relates to your comment in #601 "Refactor AggregateNetworkObjects to take TotalNetworkObjects as main argument"

Reply:
Ah. I thing I've failed to communicate what I meant there - the suggestion was that rather than taking all the arguments necessary to create a `TotalNetworkObjects` object in the `__init__` method, that this class should instead just take a `TotalNetworkObjects` object instead of those arguments.

This change actually alters the class hierarchy of the class (https://www.digitalocean.com/community/tutorials/understanding-class-inheritance-in-python-3 is a decent overview of class inheritance in python), and I imagine will cause a few issues because `TotalNetworkObjects` also inherits from `Query`, which would give you a somewhat odd inheritance hierarchy.

Reply:
OK I'll amend the code accordingly. (I do understand python class hierarchies, but I'll have a quick skim of that page)